### PR TITLE
Using ES6 packages and modules

### PR DESCRIPTION
**Pull request by jlvFlores**
On this PR, I:
- Set up a clone of my original "Awesome Books" project.
  - Created the entry point for the JavaScript code called `index.js`
  - Create d`index.css` file for the styles.
  - Created the main `index.html` file. Linking both `index.js` and `index.css` files there.
  - Created a directory called `modules`.
  - Used ES6 syntax.
  - Divided the code into **modules**. 
    - Created a separate file for each module in the `modules` directory. 
    - Imported modules into the `index.js` file.
  - Kept all functionalities of the app without errors.
  - Refactored functions to **arrow functions**.
- Display the **current date below the navigation bar**. 
  - Import it as a **module**.

### CODE REVIEW

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @jlvFlores, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project that does not have a descriptive pull request title describing the feature you are working on. KIndly modify your pull request title 👍🏽 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: modules/library.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+class Library {
+  constructor() {
+    this.books = [];
+    if (localStorage.getItem('books')) {
+      this.books = JSON.parse(localStorage.getItem('books'));
+    }
```
Comment (anagudelogu) [inline modules/library.js:6]: - [ ] **[OPTIONAL]** You can do this in one line using a fallback value, like this:

 ```suggestion
    this.books = JSON.parse(localStorage.getItem('books')) || [];
    
      
    
```
In this case, if there's no `books` key in the local storage, the value of `this.books` will fallback to an empty array `[]`. 🤓 🔧

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @jlvFlores, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/library.js (hunk 1 above)
Comment (jlvFlores) [reply modules/library.js:6]: Thank you, I knew it could be done, I just wasn't sure how to declare it. 🙏

**Pull request merged**